src/inmemory_data.jl

```diff
@@ -0,0 +1,10 @@
+struct InMemoryData
+    path::String
```

Comment:
It doesn't seem to be the right place for `path`. It's better to have a single `path` for the whole `Store`

Reply:
I'm currently use path to generate new ids for `Table`s and `Level`s. I'll remove it when I find a better solution for generating ids